.github/workflows/documentation.yaml

```diff
@@ -0,0 +1,47 @@
+on: [push, pull_request]
+name: Documentation
+jobs:
+    build:
+        name: Build
```

Comment:
Do we need both jobs (Build, doctor-rst)?

Reply:
Jepp, Doctor is only linting against some rules